### PR TITLE
Extend usage of AddCommand

### LINKED ISSUE #57: Extend behaviour of AddCommand
Instead of throwing a `ParseException` when user simply types `add` into `CommandBox`, make use of `AddWindow` to open up a new window so user can go ahead on type in there to add new `Person` into their contacts.

### PR DESCRIPTION
**Pull request by bakano98**
When a user types `add`, instead of showing an error message, it will now open up `AddWindow`.

This is similar to the behaviour of typing `help` in the `CommandBox`.

Closes #57

### CODE REVIEW

**Review by fantablack — COMMENTED** [review]
LGTM.
We can add Add Modules into the add window in another increment.

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -200,6 +200,11 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleHelp();
             }
 
+            if (commandResult.isShowAdd()) {
+                System.out.println("Detected add window");
+                handleAdd();
```
Comment (lawwm) [inline src/main/java/seedu/address/ui/MainWindow.java:205]: Remember to remove println.

**Review by lawwm — COMMENTED** [review]
LGTM

**Review by Btaykb — COMMENTED** [review]
LGTM

**Comment by bakano98** [comment]
Merging as all issues have been resolved

**Pull request merged**